apps/metadata-editor/src/app/app.routes.ts

```diff
@@ -10,100 +9,110 @@ import { NewRecordResolver } from './new-record.resolver'
 import { DuplicateRecordResolver } from './duplicate-record.resolver'
 import { AllRecordsComponent } from './records/all-records/all-records.component'
 import { MyRecordsStateWrapperComponent } from './records/my-records/my-records-state-wrapper.component'
+import { AuthGuardService } from './auth-guard.service'
 
 export const appRoutes: Route[] = [
-  { path: '', redirectTo: 'catalog/search', pathMatch: 'prefix' },
   {
-    path: 'catalog',
-    component: DashboardPageComponent,
-    outlet: 'primary',
+    path: '',
+    canActivate: [AuthGuardService],
     children: [
```

Comment:
Adding a root route (:thinking:) makes it easier to apply the same guard to all routes